indigo/distribution.yaml

```diff
@@ -4580,6 +4580,45 @@ repositories:
       url: https://github.com/GT-RAIL/jaco_gazebo.git
       version: develop
     status: maintained
+  jaguar:
+    doc:
+      type: git
+      url: https://github.com/gstavrinos/jaguar.git
+      version: kinetic-devel
+    release:
+      packages:
+      - jaguar_control
+      - jaguar_description
+      - jaguar_msgs
+      - jaguar_navigation
+      tags:
+        release: release/kinetic/{package}/{version}
```

Comment:
Please link to the exact instructions you are referring to. Otherwise or is impossible to guess which documentation you followed.

You have to run `bloom` for each ROS distribution, where each ROS distribution covers a set of Ubuntu versions (as well as Debian and Fedora). This mapping is specified in [REP 3](http://www.ros.org/reps/rep-0003.html).